dftimewolf/lib/containers/interface.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+"""The attribute container interface."""
+from __future__ import unicode_literals
+
+import six
+
+
+class AttributeContainerIdentifier(object):
+  """The attribute container identifier.
+  The identifier is used to uniquely identify attribute containers.
+  The value should be unique at runtime and in storage.
+  """
+
+  def __init__(self):
+    """Initializes an attribute container identifier."""
+    super(AttributeContainerIdentifier, self).__init__()
+    self._identifier = id(self)
+
+  def CopyToString(self):
```

Comment:
This is different from the dftimewolf style guide, which wants snake_case for method names - rename these? https://github.com/log2timeline/l2tdocs/blob/master/process/Style-guide.md